addon/behaviors/row-expansion.js

```diff
@@ -0,0 +1,23 @@
+import Behavior from 'ember-light-table/behaviors/behavior';
+
+export default Behavior.extend({
+
+  exclusionGroup: 'row-expansion',
+
+  // passed in
+  multiRow: true,
+
+  init() {
+    this._super.call(this, ...arguments);
+    this.events.onToggleExpandedRow = [ 'rowClick:_all' ];
+  },
+
+  onToggleExpandedRow(ltBody, row) {
```

Comment:
Is there a reason you are passing the entire ltBody component here and not just the table instance?